### ISSUE TITLE
Network Burn Address does not work

### ISSUE THREAD

**Issue by mitchellcash** [PaycoinFoundation/paycoin#318]
When attempting to send coins to the Mainnet Network Burn Address you get a lot of strange errors and the transaction does not go through:

**Comment by IngCr3at1on** [comment]
Yeah someone else mentioned it not working also; iirc I probably should have only pushed the testnet one but being that the usage required a fork I was hoping not to have to have to fork multiple times for it. I'm going to look at it on the testnet and try to figure out what's going on though.

**Comment by IngCr3at1on** [comment]
Was looking at this yesterday and what's odd is that it works on the testnet without issues.

**Comment by mitchellcash** [comment]
Agreed. I even tested it myself on the testnet and its fine. So what's different about testnet and mainnet

**Comment by IngCr3at1on** [comment]
Figured out what was going on. Burn transactions are non-standard transactions which aren't accepted on the mainnet. I might have figured out a decent solution but sadly not in time for the release that everyone had agreed on so it will be awhile before the fix is actually deploy-able.

**Comment by mitchellcash** [comment]
Annoying we couldn't get the fix into this release but I look forward to this in the next release.

**Comment by IngCr3at1on** [comment]
Well personally I'd like the next release to not be a fork but rather just general updates. So maybe another release after.
